include/message_filters/subscriber.h

```diff
@@ -52,22 +53,34 @@ class SubscriberBase
    *
    * If this Subscriber is already subscribed to a topic, this function will first unsubscribe.
    *
-   * \param node The rclcpp::Node::SharedPtr to use to subscribe.
+   * \param node The rclcpp::Node::SharedPtr or rclcpp_lifecycle::LifeCycleNode::SharedPtr  to use to subscribe.
    * \param topic The topic to subscribe to.
    * \param qos (optional) The rmw qos profile to use to subscribe
    */
-  virtual void subscribe(rclcpp::Node::SharedPtr node, const std::string& topic, const rmw_qos_profile_t qos = rmw_qos_profile_default) = 0;
-
+  // [[deprecated("Use rclcpp::QoS to specify qos")]]
```

Comment:
Sorry it took so long for me to get back to this, but why are these commented out now?